app/controllers/conference_controller.rb

```diff
@@ -21,10 +25,40 @@ def schedule
     conf_end = 20
     @conf_period = conf_end - @conf_start
 
-    if @dates == Date.current
-      @today = Date.current.strftime('%Y-%m-%d')
+    @day = Time.find_zone(@conference.timezone).today
+    # the schedule takes you to today if it is a date of the schedule
+    if @dates.include? @day
+      @has_date = true
+      current_hour = DateTime.now.in_time_zone(@conference.timezone).strftime('%H').to_i
```

Comment:
If the conference has finished and someone enter to see the schedule I think that we should render the next day.

I have to consider that case even if I do other thing. :confused: